### PR TITLE
Update the manual testing appendix of DG

### LINKED ISSUE #239: DG: Placeholder in launch and shutdown
Screenshot 2023-11-11 at 10 19 33 PM

### PR DESCRIPTION
**Pull request by Tim-Siu**

### CODE REVIEW

File: src/main/java/seedu/address/logic/Messages.java [hunk 1]
```diff
@@ -18,8 +18,8 @@ public class Messages {
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid!";
     public static final String MESSAGE_INVALID_COMPANY_DISPLAYED_INDEX = "The company index provided is invalid!";
-    public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
-    public static final String MESSAGE_COMPANIES_LISTED_OVERVIEW = "%1$d companies listed!";
+    public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d person(s) listed!";
+    public static final String MESSAGE_COMPANIES_LISTED_OVERVIEW = "%1$d company(ies) listed!";
```
Comment (lunaroddity) [inline src/main/java/seedu/address/logic/Messages.java]: Will this change the output of the UI?

Comment (Tim-Siu) [reply src/main/java/seedu/address/logic/Messages.java]: This will change the output. Not sure if this is allowed

Comment (lunaroddity) [reply src/main/java/seedu/address/logic/Messages.java]: Following what PRof has said, I don't think we are allowed to change this

Comment (Tim-Siu) [reply src/main/java/seedu/address/logic/Messages.java]: I reverted the modification

**Pull request merged**